api/messengertypes.proto

```diff
@@ -417,6 +407,14 @@ message Interaction {
   bool acknowledged = 10;
   string target_cid = 13 [(gogoproto.moretags) = "gorm:\"index;column:target_cid\"", (gogoproto.customname) = "TargetCID"];
   repeated Media medias = 15;
+
+  // specific to client model
+  repeated ReactionView reactions = 16 [(gogoproto.moretags) = "gorm:\"-\""];
```

Comment:
Name should be changed imo, view is used in too many contexts already

Reply:
open to proposals ^^